opacus/validators/__init__.py

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 
-from .batch_norm import validate, fix  # noqa
-from .ddp import validate, fix  # noqa
-from .instance_norm import validate, fix  # noqa
-from .lstm import validate, fix  # noqa
+from .batch_norm import fix, validate  # noqa
+from .ddp import fix, validate  # noqa
+from .instance_norm import fix, validate  # noqa
+from .lstm import fix, validate  # noqa
```

Comment:
What was the reason of doing this?
I would wrap this in a function to make it self-explanatory. Otherwise isort will mix everything together.

```python
from .module_validator import ModuleValidator
from .utils import register_module_validator, register_module_fixer

def check_validator_submodules():
    from .batch_norm import fix, validate  # noqa
    from .ddp import fix, validate  # noqa
    from .instance_norm import fix, validate  # noqa
    from .lstm import fix, validate  # noqa
    from .multihead_attention import fix, validate  # noqa

check_validator_submodules()

__all__ = [
    "ModuleValidator",
    "register_module_validator",
    "register_module_fixer",
]

```

Reply:
@karthikprasad wdyt?

I think the purpose of this is to make sure `register_validator` methods are called - and I'm not sure what's the best way to make it obvious for the reader here

Reply:
> I think the purpose of this is to make sure register_validator methods are called 

That is correct. I don't know if there is an accepted convention.

> and I'm not sure what's the best way to make it obvious for the reader here

Perhaps add Igor's explanation as a comment on the top of the file explaining the need for such imports?

> I would wrap this in a function to make it self-explanatory.

Wrapping in a function is also fine, but I don't think it is self-explanatory. A comment might still be necessary, in which case, we can do away with the function and simply use a comment? WDYT?

> Otherwise isort will mix everything together.

What is the issue here? The order of import doesn't matter as long as all are imported.

Reply:
A comment is a must for the future contributors to understand why it is necessary to import submodules. Igor's explanation is good enough. Thanks, now I know this too :)

> What is the issue here? The order of import doesn't matter as long as all are imported.

The order doesn't matter for the lib to work, but is does matter for the human to read. The way the code structured is like a docstring.

> we can do away with the function and simply use a comment? 

Agree, but now all imports are mixed up, there is no good place to put explanation, look:

```python
from .lstm import fix, validate  # noqa
from .module_validator import ModuleValidator
from .multihead_attention import fix, validate  # noqa
from .utils import register_module_fixer, register_module_validator
```

If all `register_validator`-related imports are in one place it is more clear for the contributor that they need to put the module here after adding a new validator.

Reply:
> Agree, but now all imports are mixed up, there is no good place to put explanation, look:

Ouch! Ugly!

> If all register_validator-related imports are in one place it is more clear for the contributor that they need to put the module here after adding a new validator.

+3000